dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/internal/statemachine/StartAction.java

```diff
@@ -128,13 +134,11 @@ public void execute(ExecutorService executor) {
             @SuppressWarnings("deprecation")
             @Override
             public void run() {
-                AppMessageHelper.get(_aii.getHandler()).audit("APPLICATION_SLOW_STARTUP", _config.getName(), TimestampUtils.getElapsedTime(_startTime.get()));
+                AppMessageHelper.get(handler).audit("APPLICATION_SLOW_STARTUP", _config.getName(), TimestampUtils.getElapsedTime(_startTime.get()));
             }
         }, maxWait, TimeUnit.SECONDS));
 
         try {
-            @SuppressWarnings("rawtypes")
-            ApplicationHandler handler = _aii.getHandler();
             @SuppressWarnings("unchecked")
             ApplicationMonitoringInformation ami = handler.setUpApplicationMonitoring(_aii);
```

Comment:
I don't see how an NPE is prevented here when `handler==null`. Did you intend to return early from the `if (handler == null)` check above?